.github/workflows/pruner.yml

```diff
@@ -0,0 +1,12 @@
+name: Openstack Pruner
+run-name: Openstack Pruner
+on:
+  schedule:
+    - cron:  '*/5 */1 * * *'
```

Comment:
we're going to run this every 5 minutes???

Reply:
“At every 5th minute past every hour.”

Reply:
*/5 means every 5 minutes
*/1 means every one hour

Reply:
Good geez this is why coding on 3 hours of sleep is a bad idea.  fixing